fixtures/provider.py

```diff
@@ -110,8 +110,8 @@ def template(template_location, provider):
     pytest.skip('Template not available')
 
 
-def _small_template(provider):
-    template = provider.data.get('small_template', None)
+def _get_template(provider, template_name):
+    template = provider.data.get(template_name, None)
```

Comment:
`template_name` is not really a template name, it's a template type name so we should call it that.